Makefile

```diff
@@ -17,6 +17,7 @@ LOCAL_DIRS            = $(shell govendor list -p -no-status +local)
 TEST_FLAGS            ?= -v -parallel 32
 TEST_OUTPUT           ?= test.xml
 TEST_DOCKER_OUTPUT    ?= test-standalone-docker.xml
+GOVENDOR              := $(GOPATH)/bin/govendor
```

Comment:
This is pretty neat.

Reply:
Is there a way we can ensure that govendor is at least version X? Maybe have it depend on vendor/vendor.json, and make it go get -u ...?

Reply:
Good question - the best I can think of would involve using something like https://github.com/cloudflare/semver_bash to check its version.  I'll probably leave that to another PR, unless I can think of something simpler.